ansible/roles/redpanda/tasks/install_redpanda.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: add the redpanda repo
+  shell: |
+    curl -1sLf https://packages.vectorized.io/sMIXnoa7DK12JW4A/redpanda/cfg/setup/bash.rpm.sh | sudo -E bash
+  args:
+    warn: no
+
+- name: install redpanda from repository
+  package:
+    name:
+      - redpanda
+    state: present
+    update_cache: yes
+
+- name: set data dir file perms
+  file:
+    path: /var/lib/redpanda/data
```

Comment:
is this on the mounted GP2 SSD drive, or on instnace storage? I never can remember where this ends up mounted :)

Reply:
ah it's linked in prepare storage, yes?